pom.xml

```diff
@@ -173,6 +173,19 @@
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.21.0</version>
+        <configuration>
+          <includes>
+            <include>**/Test*.java</include>
+            <include>**/*Test.java</include>
+            <include>**/*Tests.java</include>
+            <include>**/*TestCase.java</include>
+          </includes>
+        </configuration>
+      </plugin>
```

Comment:
Since Maven invokes Surefire by default and you're not providing a custom goal here, it's slightly nicer to move this change up into the `<pluginManagement>` section, next to the `maven-compiler-plugin` and `maven-release-plugin` :)

Reply:
Ah okay, thanks for the feedback @Stephan202! I'll submit it as a new PR.

Reply:
Done. :)